### PR TITLE
Added sample skeleton apps for Java, ASPNET, and Node.js

### PR DESCRIPTION
**Pull request by amgrobelny-box**

### CODE REVIEW

File: box-node-express-skeleton-app/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    boxClientId = Your Client ID
+    boxClientSecret = Your Client Secret
+    boxPrivateKey = Your private key, stored in the root of `box-aspnet-mvc-skeleton-app`
+    boxPublicKeyId = Your Public Key ID
+    boxPrivateKeyPassword = Your Private Key Passphrase
+    boxEnterpriseId = Your Enterprise ID
+    ```
+
+####Auth0 Configuration
+Additionally, since you manage the identity and authorization for your Box App Users within your ASPNET MVC application, you'll need an identity service to fully utilize JWT authentication on behalf of your App Users.
```
Comment (mself) [inline box-node-express-skeleton-app/README.md]: Looks like cut and paste reference to ASPNET in the Node readme

**Review by mself — CHANGES_REQUESTED** [review]

**Comment by amgrobelny-box** [comment]
Nice catch! Definitely a copy/paste error.

File: box-node-express-skeleton-app/LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (mself) [inline box-node-express-skeleton-app/LICENSE]: I think we're supposed to use Apache-2.0

File: box-node-express-skeleton-app/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "version": "1.0.0",
+  "description": "A skeleton Express app with authentication and Box Platform App Users",
+  "main": "bin/www",
+  "private": true,
+  "scripts": {
+    "prestart": "npm run test",
+    "dev-start": "./node_modules/.bin/gulp develop",
+    "start": "node ./bin/www"
+  },
+  "author": "AMSXBG",
```
Comment (mself) [inline box-node-express-skeleton-app/package.json]: Can you switch to:
  "author": "Box <oss@box.com>",
  "license": "Apache-2.0"

**Review by mself — CHANGES_REQUESTED** [review]

**Comment by amgrobelny-box** [comment]
Made various changes, including those outlined here and that we discussed on chat!

**Comment by KarthikShanmugasundaram** [comment]
Matthew Self review comments were addressed.

**Pull request merged**